packages/explat-client/src/create-explat-client.ts

```diff
@@ -65,6 +65,12 @@ export function createExPlatClient( config: Config ): ExPlatClient {
 		throw new Error( 'Running outside of a browser context.' );
 	}
 
+	const safeLogError: typeof config.logError = ( ...args ) => {
+		try {
+			config.logError( ...args );
+		} catch ( e ) {}
+	};
```

Comment:
The code in this patch looks exceedingly defensive. Is it common for `config.logError` to throw exceptions? I see that the actual implementation we pass in the config object already handles inner exceptions:

https://github.com/Automattic/wp-calypso/blob/trunk/client/lib/explat/internals/log-error.ts

Reply:
Ah I just had to move the `safeLogError` higher in the code here.

By design `loadExperiment` is never supposed to fail, it is not likely for `logError` to fail but this gives us a guarantee. Additionally this package will be used by other platforms with different `logError` implementations.